batchflow/models/torch/base.py

```diff
@@ -316,6 +316,7 @@ def __init__(self, config=None):
         self.devices = []
 
         # Train procedure and ifrastructure
+        self.init_model_weights = None
```

Comment:
mention that it is a kaiming normal initialization. Rename to `kaiming_normal_init` or etc.

Reply:
Ok, the method is renamed to ```init_kaiming_normal``` (like torch style), the flag renamed to ```kaiming_normal_init```.

Reply:
Also, I added the ```init_zero_bias``` flag to separate these initializations.